semantic_release/cli/commands/cli_context.py

```diff
@@ -58,7 +58,7 @@ def _init_runtime_ctx(self) -> RuntimeContext:
 
         try:
             if was_conf_file_user_provided and not conf_file_exists:
-                raise FileNotFoundError(
+                raise FileNotFoundError(  # noqa: TRY301
```

Comment:
@codejedi365: it _looked_ to me like this could maybe go outside the try / except, but didn't want to mess with it here?

Reply:
No, its purposefully inside the try/except so that the `click` context is properly exited cleanly.